pcdsdevices/lxe.py

```diff
@@ -414,6 +413,165 @@ def format_status_info(self, status_info):
 """
 
 
+class Lcls2LaserTiming(FltMvInterface, PVPositioner):
+    """
+    A "lxt" motor for the LCLS-II laser locker, AKA the OPCPA laser locker.
+    """
+
+    tab_component_names = True
+
+    verbose_name = 'Laser X-ray Timing'
+
+    _tgt_time = Cpt(EpicsSignal, ':PHASCTL:DELAY_SET', auto_monitor=True,
+                    kind='omitted',
+                    doc='The internal nanosecond-expecting signal.'
+                    )
+    setpoint = Cpt(_ScaledUnitConversionDerivedSignal,
+                   derived_from='_tgt_time',
+                   derived_units='s',
+                   original_units='ns',
+                   kind='hinted',
+                   doc='Setpoint which handles the timing conversion.',
+                   limits=(-100e-6, 100e-6),
```

Comment:
This is a good thing to think about. Off-hand, I think there really isn't something we can determine easily _a priori_, in part because the limit would be related to laser rep-rate, which could change even during the course of a single experiment. 
The 100us limits seemed OK to me, since they're wide enough for almost all experiments, and can be modified on the fly if truly necessary.